src/content/docs/en/chromebook/faq-and-troubleshooting.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Troubleshooting and FAQs
+description: "Troubleshooting tips and tricks, and FAQs"
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+## Troubleshooting
+
+#### Chromebook fails to boot to internal disk
```

Comment:
why going so far with titles? why not 3?

Reply:
why #### instead of ###

Reply:
Ah, eh. Might change thought it looked better.